bindings/python/test/environment/magnetic/earth/test_manager.py

```diff
@@ -19,8 +14,6 @@
 def manager() -> EarthMagneticModelManager:
     manager = EarthMagneticModelManager.get()
 
-    manager.set_local_repository(EarthMagneticModelManager.default_local_repository())
```

Comment:
So the reason I removed that is because when you instantiate the manager, it by default sets it to the default local repository, so it felt a bit redundant.